_data/sitetext.yml

```
@@ -62,9 +62,9 @@ money:
         - text: Cloud services
           img: fall_slow
           desc: Data storage and computation power for the recommendation algorithms.
-        - text: Initial data sources
+        - text: Development costs
           img: fall_slow
-          desc: This may or may not be a cost. In order to offer recommendations to the first users we will need data, before we gather data from their learning journals.
+          desc: Developer salaries.
```

```suggestion
          desc: Developer salaries, equipment, development tools and other expenses related to the development of the product.
```

Let's include a bit more details in the description.